pom.xml

```diff
@@ -106,6 +106,21 @@
       <version>4.0.3.Final</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>build.buf.gen</groupId>
```

Comment:
I might not be understanding everything at a glance, but if these dependencies are pulling in the generated code from maven, then we shouldn't need to add the generated code to this repo. We can discuss in the collab.

Reply:
So we can remove the generated code as it does exist in maven with imports `build.buf.gen.kessel.relations.v1beta1` and `build.buf.gen.kessel.relations.v1`. 

Would probably also want to remove `option java_package = "org.project_kessel.api.relations.v1";` from the proto files as it doesn't use this path anymore.

Reply:
I would have thought `java_package` would also be used by the buf tooling to generate the classes, because the package spec can't be taken from anywhere else. Or are you specifying it in an external buf console somewhere? 

In any case, I would keep it and maintain it since the proto spec should be the source of truth for all the code generation for the clients.

Reply:
It seems that buf is using the package name directly from the proto files, like its specified here: https://github.com/project-kessel/relations-api/blob/68cda88c91435e7696c565a680311f508ccf2aa4/api/health/v1/health.proto#L3